.github/workflows/chewbbaca.yml

```diff
@@ -2,17 +2,17 @@ name: chewbbaca
 
 on:
   push:
-    branches: [ main ]
+    branches: [ dev_SchemaEvaluator ]
   pull_request:
-    branches: [ main ]
+    branches: [ dev_SchemaEvaluator ]
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7]
```

Comment:
We should test with several Python versions. Maybe change it to [3.6, 3.7, 3.8]?

Reply:
Don't forget the changes to dictionary structures in python 3.7 that will break many of the tests.